etl/steps/data/garden/technology/2024-05-13/computer_memory_storage.py

```diff
@@ -21,7 +21,7 @@ def run(dest_dir: str) -> None:
     #
     # Process data.
     #
-    us_cpi = catalog.find_latest(dataset="us_consumer_prices").reset_index()[["year", "all_items"]]
+    us_cpi = ETLCatalog().find_latest(dataset="us_consumer_prices").reset_index()[["year", "all_items"]]
```

Comment:
We shouldn't be using `ETLCatalog` in steps. This should be a proper dependency and loaded via `paths.` or `Dataset(`.